wp-content/themes/core/pcss/base/_fonts.pcss

```diff
@@ -16,3 +16,59 @@
 .visual-tests * {
 	font-family: arial, sans-serif !important;
 }
+
```

Comment:
Let's go ahead and add the comment doc block for each font family and `visual-tests *` can go at the bottom of this file ; )